src/style/blocks/page.css

```diff
@@ -0,0 +1,17 @@
+@font-face {
+  font-family: Avenir;
+  src: url(../../fonts/Avenir.ttc);
+}
+
+html,
+input {
```

Comment:
```suggestion
html {
```
<img width="1153" alt="Знімок екрана 2024-08-14 о 15 16 04" src="https://github.com/user-attachments/assets/640c50d0-e09c-4cbe-ae4e-a1dda8771786">